buildpipeline/dotnet-standard-win.json

```diff
@@ -107,7 +107,7 @@
       },
       "inputs": {
         "filename": "build.cmd",
-        "arguments": "-- publish.msbuild /t:PublishToAzureBlobFeed /p:AccountKey=$(PB_PublishAccountKey) /p:ExpectedFeedUrl=$(PB_AzureFeedUrl)",
+        "arguments": "-- publish.msbuild /t:PublishToAzureBlobFeed /p:AccountKey=$(PB_PublishAccountKey) /p:ExpectedFeedUrl=$(PB_AzurePublishFeedUrl)",
```

Comment:
What made you change to this to PB_AzurePublishFeedUrl? I'm currently working on some documents about these exact properties and I've tentatively named it PB_BlobFeedUrl and PB_BlobFeedKey. Although I'm open to suggestions that is what I'm about to push all repo's to use so please go ahead and use those here as well.

Reply:
Listening to the prod con meeting this morning, and after discussion with @mmitche , I realized that we needed to be able to define the consumption feed and the publish feed separately, they may not be the same feeds.

Reply:
Hmm, seeing it here, I think I'd like having `Publish` somewhere in the name (or some word like it) to make it obvious at first glance that it isn't related to package restore. `PB_PublishBlobFeed[Url/Key]`, `PB_OutputBlobFeed[Url/Key]`?

Reply:
I'll hold off on this PR until I see the doc, unless it's going to be more than a day.  If it's going to be more than a day to settle things, then I'll just merge this and follow up once the naming has solidified.

Reply:
I like using publish, I've updated the document. 

@chcosta use PB_PublishBlobFeedUrl and PB_PublishBlobFeedKey.

Reply:
Do we need separate variables for `PB_PublishBlobFeedUrl` and `PB_PublishBlobSymbolFeedUrl`?  I saw this practice of separating the feeds in a different PR and copied the model here.

Reply:
No lets not have a separate on for symbols at this point.